bigquery/data-transfer/api/QuickStart/QuickStart.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * Copyright (c) 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+// [START bigquery_data_transfer_quickstart]
+
+using System;
+using Google.Api.Gax;
+using Google.Cloud.BigQuery.DataTransfer.V1;
+
+namespace GoogleCloudSamples
+{
+    public class QuickStart
+    {
+        public static void Main(string[] args)
+        {
+            // Instantiates a client
+            DataTransferServiceClient client = DataTransferServiceClient.Create();
+
+            // Your Google Cloud Platform project ID
+            string projectId = "YOUR-PROJECT-ID";
```

Comment:
please, add an explicit check that will throw exception if user runs your sample straight away and forgets to replace projectId value with real value.

Reply:
That would complicate the sample (much undesired for a "quickstart") and is inconsistent with other quickstart samples in this repo such as https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/master/pubsub/api/QuickStart/QuickStart.cs